examples/testbed/ui.rs

```diff
@@ -882,3 +885,96 @@ mod radial_gradient {
+                    ),
+                    (
+                        UiTransform::from_scale(Vec2::new(2., 0.5)),
+                        "Scale 2x 0.5.y",
+                        GREEN,
+                    ),
+                    (
+                        UiTransform::from_translation(Val2::px(-50., 50.)),
+                        "Translate -50px x +50px y",
+                        BLUE,
+                    ),
+                    (
+                        UiTransform {
+                            translation: Val2::px(50., 0.),
+                            scale: Vec2::new(-1., 1.),
+                            rotation: Rot2::degrees(30.),
+                        },
+                        "T 50px x\nS -1.x (refl)\nR 30d",
+                        DARK_CYAN,
+                    ),
+                ] {
+                    parent
+                        .spawn((
+                            Node {
+                                width: percent(100),
+                                margin: UiRect {
+                                    top: px(50),
+                                    bottom: px(50),
+                                    ..default()
+                                },
+                                align_items: AlignItems::Center,
+                                justify_content: JustifyContent::SpaceAround,
+                                ..default()
+                            },
+                            DespawnOnExit(super::Scene::Transformations),
+                        ))
+                        .with_children(|row| {
+                            row.spawn((
+                                Text::new("Before Tf"),
+                                Node {
+                                    width: px(100),
+                                    height: px(100),
+                                    ..default()
+                                },
+                                BackgroundColor(background.into()),
+                                TextFont {
+                                    font: asset_server.load("fonts/FiraSans-Bold.ttf"),
+                                    ..default()
+                                },
+                                DespawnOnExit(super::Scene::Transformations),
+                            ));
+                            row.spawn((
+                                Text::new(label),
+                                Node {
+                                    width: px(100),
+                                    height: px(100),
+                                    ..default()
+                                },
+                                BackgroundColor(background.into()),
+                                transformation,
+                                TextFont {
+                                    font: asset_server.load("fonts/FiraSans-Bold.ttf"),
+                                    ..default()
+                                },
```

Comment:
```suggestion
                                TextFont::default(),
```
using the default font removes the risk for an asset loader delay